packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -11,7 +11,7 @@ interface DataRowProps<T extends DomainObject> {
   onClick?: (rowData: T) => void;
   rowData: T;
   rowKey: string;
-  ExpandContent?: FC;
+  ExpandContent?: FC<{ rowData: T }>;
```

Comment:
Can remove this if you'd rather it not be on the demo server, but I just stringified the summary item and displayed it in the expando for something to look at